website/src/css/custom.css

```diff
@@ -2143,4 +2143,66 @@ table th {
 [data-theme='dark'] table th {
   color: #000000; /* Black text on darker background */
   font-weight: bold;
-}
\ No newline at end of file
+}
+
+.tabs {
+  background-color: transparent;
+  padding: 10px;
+}
+
+/* Default color and border for tabs */
+.tabs__item {
+  background-color: #f9fbfc; /* Sets the tab color the same as the file borders */
+  border: 2px solid #ccc;
+  padding: 5px 10px;
+  margin-right: 5px;
+  border-radius: 4px;
+  cursor: pointer;
+  transition: background-color 0.3s;
+}
+
+/* Default color and border for tabs in dark mode */
+[data-theme='dark'] .tabs__item {
+  background-color: #000000;
+  border: 1px solid #ccc;
+  padding: 5px 10px;
+  margin-right: 5px;
+  border-radius: 4px;
+  cursor: pointer;
+  transition: background-color 0.3s;
+}
+
+/*Color when hovering over tabs */
+.tabs__item:hover {
+  background-color: #D95B3B;
```

Comment:
`#d95b3b` is not an available brand color. Could we swap this with `#DE5D43` or `#ff694a`? `#ff694a` has a variable already, but a new variable would need to be created for `#DE5D43` if you go with that one.

Reply:
Also, on hover, the lighter text is still set on the tab while the orange background is applied. This doesn't pass the WCAG accessibility check for contrast [link here](https://webaim.org/resources/contrastchecker/). The text color may need to be updated on hover to match the color when the tab is active.
![image](https://github.com/user-attachments/assets/20851b6a-6d40-4c50-bcc8-24a4eac6db45)

Reply:
And on hover, can we have the border color update to match the background color as well? Similar to how the border updates for the active tab.